src/harness/renderers/gl/gl_renderer.c

```diff
@@ -500,6 +500,11 @@ void setActiveMaterial(tStored_material* material) {
         glUniform1i(uniforms_3d.light_value, -1);
     }
 
+    if (material->flags & BR_MATF_TWO_SIDED)
+        glDisable(GL_CULL_FACE);
+    else
+        glEnable(GL_CULL_FACE);
+
```

Comment:
```suggestion
    if (material->flags & BR_MATF_TWO_SIDED || material->flags & BR_MATF_ALWAYS_VISIBLE) {
        glDisable(GL_CULL_FACE);
    } else {
        glEnable(GL_CULL_FACE);
    }
```

Reply:
there is another flag `BR_MATF_ALWAYS_VISIBLE` that we should probably treat the same way